bashbrew/bashbrew.sh

```diff
@@ -97,10 +97,9 @@ done
 
 # which subcommand
 subcommand="$1"
+shift || { usage >&2 && exit 1; }
 case "$subcommand" in
-	build|push|list)
-		shift
```

Comment:
This one led to fun messages like `unknown subcommand: ` when you don't supply any arguments at all.